langchain4j/src/test/java/dev/langchain4j/service/AiServicesWithChatMemoryIT.java

```diff
@@ -242,7 +243,7 @@ void should_keep_chat_memory_and_add_new_system_message() {
     }
 
 
-    interface ChatWithSeparateMemoryForEachUser {
+    interface ChatWithSeparateMemoryForEachUser extends ChatMemoryAccess {
```

Comment:
Extending the `ChatMemoryAccess` interface (I'm not strongly opinionated on the name) allows to access the `ChatMemory` of the AI service.